package/src/Platform/Platform.web.tsx

```diff
@@ -116,12 +117,16 @@ const View = (({ children, onLayout, style: rawStyle }: ViewProps) => {
 
 export const Platform: IPlatform = {
   OS: "web",
-  PixelRatio: window.devicePixelRatio,
+  PixelRatio: (typeof window !== 'undefined') ? window.devicePixelRatio : 1, // accounting for Metro bundler that might have an undefined `window` value
```

Comment:
this change is very sensible can you make it as a separate PR so we can merge it quickly?